planner/core/expression_rewriter_test.go

```diff
@@ -260,3 +260,19 @@ func (s *testExpressionRewriterSuite) TestPatternLikeToExpression(c *C) {
 	tk.MustQuery("select 0 like '0';").Check(testkit.Rows("1"))
 	tk.MustQuery("select 0.00 like '0.00';").Check(testkit.Rows("1"))
 }
+
+func (s *testExpressionRewriterSuite) TestFetchNullData(c *C) {
```

Comment:
TestFetchNullData -> TestIssue16679, and please add a comment `// TestIssue16679 contains tests for https://github.com/pingcap/tidb/issues/16679.`